predicthq/endpoints/schemas.py

```diff
@@ -141,6 +141,13 @@ class Location(StringModel):
     longitude = FloatType(required=True)
 
 
+class Parent(Model):
+    class Options:
+        serialize_when_none = False
+
+    include = StringType()
```

Comment:
In order to be consistent with how the SDK works, it would be a good thing for the `parent.include` field to validate the data it is being assigned (`bool` or `Enum["only"]`).

That being said, we have a task in our backlog to get rid of the SDK validation and let the API handle it (in order to avoid duplication).
Could you provide an example of what the ClientError would look like when the `search` method gets used with an incorrect value for `parent.include`? If good enough, we probably can go with this.

Reply:
Great collab on this 💪🏻 thank you for the help in understanding the problem and for the work on getting this in a much better place! 

Addressed in [e7898e7](https://github.com/predicthq/sdk-py/pull/84/commits/e7898e749338a82719df3392270d5fcec0da2bc3)